src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/msg/ErrorMsg.java

```diff
@@ -27,7 +28,8 @@ public ErrorMsg(
         @JsonProperty("poBox") String poBox,
         @JsonProperty("documentControlNumber") String documentControlNumber,
         @JsonProperty(value = "errorCode", required = true) ErrorCode errorCode,
-        @JsonProperty(value = "errorDescription", required = true) String errorDescription
+        @JsonProperty(value = "errorDescription", required = true) String errorDescription,
+        @JsonProperty(value = "service", required = true) String service
```

Comment:
should not be required yet. There might be messages on the queue on some environments.

Reply:
updated to make service optional